js/build_map.js

```diff
@@ -1,14 +1,43 @@
-// Width and height
-var chart_width     =   1000;
-var chart_height    =   700;
-
-// define categories
+// Define one big global to eventually rule them all.
```

Comment:
👑 I like this idea! Still let's us have globals, but less of a chance for having namespace var name conflicts and such.

Reply:
good! I believe this is a pretty standard way of doing namespacing in javascript, so it gets us a start on #127